lib/bindings/AMQPBinding.js

```diff
@@ -78,6 +78,20 @@ function start(callback) {
         queue = constants.AMQP_DEFAULT_QUEUE;
```

Comment:
CNR entry needed?

Probably not, but just checking...

Reply:
CNR is included in previous PR